activity/kafkapub/ui/activity.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "tibco-kafkapub",
+  "version": "0.0.1",
+  "type": "flogo:activity",
+  "ref": "github.com/wnichols/flogo-contrib/activity/kafkapub",
```

Comment:
Changed them all.  I had a script that switches them and I forgot to run it.  Is there a convenient way to test a contribution to flogo without switching all the url's?